solidity/.gitignore

```diff
@@ -6,6 +6,7 @@ dist/
 coverage/
 coverage.json
 .env
+*lcov.base.info
```

Comment:
should this include *lcov.info also

Reply:
Doesn't matter, I don't think we write *lcov.info to ./soldiity